src/download_dataset.py

```diff
@@ -0,0 +1,57 @@
+import gdown
+import tarfile
+import os
+import glob
+from pathlib import Path
+import argparse
+
+# If the download doesn't work, you can download the dataset manually from the following link:
```

Comment:
Az ilyen kommenteket amúgy dobjuk be egy doksiba `doc/` directoryba. Ez mondjuk hetene a `doc/data_acquisition.md` fájlba 😄